### DISCUSSION TITLE
Missing Values Codelist main-activity_entity-activity.gc

### DISCUSSION THREAD

**Discussion by JanEmde** [OP-TED/eForms-SDK#925] [Q&A] [answered]
We have noticed that the field ‘The main activity of the contracting entity’ has been mandatory in certain cases in Directive 2014/23/EU forms since SDK 1.8. For example in subtypes 14 and 19. 

However, it appears that the content of the associated codelist (main-activity_entity-activity.gc)  is still only tailored to Directive 2014/25/EU. In our view, for example, an entry for the award of a concession for catering services is missing here.

For example, no valid entry can be selected in the following notice, although an e-application is mandatory from SDK 1.8:
https://ted.europa.eu/de/notice/-/detail/191848-2024

**Comment by YvesJo** [comment]
Hi,
Could you please specify which code should be in the entity-activity tailored list?

**Reply by JanEmde** [reply]
I don't know exactly what is conceivable, but in this case the awarding authority is a science and congress centre (Public company controlled or financed by a local authority) that awards a concession for catering services.

However the codeliste contains only activities regarding transport, water and energy supply (Directive 2014/25/EU).

**Reply by YvesJo** [reply, marked as answer]
Thanks for these details Postal services are also there.
I'm wondering whether that organization has not improperly been classified as Contracting Entity instead of Contracting Authority, you'll be able to confirm.

Following Art 2 of D24:

> ‘**contracting authorities**’ means the State, regional or local authorities, **bodies governed by public law** or associations formed by one or more such authorities or one or more such bodies governed by public law;

and additionally, defined in the same article:

> ‘**bodies governed by public law**’ means bodies that have all of the following characteristics:
- they are established for the specific purpose of meeting needs in the general interest, not having an industrial or commercial character;
- they have legal personality; and
- they are financed, for the most part, by the State, regional or local authorities, or by other bodies governed by public law; or are subject to management supervision by those authorities or bodies; or have an administrative, managerial or supervisory board, more than half of whose members are appointed by the State, regional or local authorities, or by other bodies governed by public law;

**Reply by JanEmde** [reply]
Thank you for the information. We will clarify this with the awarding authority.

**Reply by JanEmde** [reply]
@YvesJo Unfortunately, we did not receive any response back then. As we now have a new request on this topic, I would like to revisit it:

1. Postal services are also covered by Directive 2014/25/EU. This therefore underlines the fact that only values from the directive are listed in the code list.
2. The organization was classified as “pub-undert-ra”. That sounds like the right classification to me.

**Reply by ferrakaop** [reply]
Please see the ongoing discussion with Member States about the activity codelists: https://code.europa.eu/eproc/eforms/crs/-/issues/4

We will implement whatever gets decided there (after the 3rd amendment so in SDK 2.1. next year).

**Reply by JanEmde** [reply]
Thanks for the information!
